src/interaction-handlers/transfer.ts

```diff
@@ -17,10 +13,12 @@ export class TransferHandler extends InteractionHandler {
     });
   }
   // Get the game info and the interaction type
-  public override parse(interaction: ButtonInteraction): Maybe<{
-    transferId: string;
-    sign: TransferSign;
-  }> {
+  public override parse(interaction: ButtonInteraction):
```

Comment:
In Sapphirev4, Maybe no longer exists. We use Option.None and Option.Some instead.